go.work

```diff
@@ -0,0 +1,6 @@
+go 1.20
+
+use (
+    .
+    ./conformance/echoserver
```

Comment:
@robscott I wonder if we need to use more clear names for the echo servers, given that we have two of them. This one is just an http echo server (that echos the request params) vs. the other mesh echo server which supports various protocols (and echos the request body)?

Reply:
Agreed, maybe we can just call them `echo-ingress` and `echo-mesh` for now?

Reply:
I think we should leave the _current_ use out, and call them `echo-basic` and `echo-advanced` or something instead. It would be weird if we end up using `echo-mesh` for the ingress tests, but `echo-advanced` is fine.